src/metabase/actions/execution.clj

```diff
@@ -114,37 +114,39 @@
         extra-parameters         (set/difference (set (keys request-parameters))
                                                  (set (keys slug->field-name)))
         pk-field                 (first pk-fields)
-        simple-parameters        (update-keys request-parameters (comp keyword slug->field-name))
+        simple-parameters        (->> (update-keys request-parameters (comp keyword slug->field-name))
+                                      (filter (fn [[_k v]] (some? v)))
```

Comment:
we probably should have a comment explaining this here.

Reply:
@qnkhuat done